src/components/CommentsList.jsx

```diff
@@ -0,0 +1,12 @@
+import CommentCard from "./CommentCard";
+
+export default function CommentsList({ comments }) {
```

Comment:
an isloading would be really useful here, or consideration for what would appear if there are no comments